index.html

```diff
@@ -34,6 +34,155 @@ <h1>Charlie<br>Giorgania</h1>
   <img src="svg/image_geometry_2.svg" class="blockImage" alt="">
 </div>
 
+<section>
+  <header>
+    <h1>Projects</h1>   
+  </header>
+  <div>
+    <figure> 
+      <img src="Images/project-image-1.jpg" class="blockImage" alt="">
+    </figure>
```

Comment:
- [ ] For decorations, it is better to use  background images (`background-image` CSS property) instead of using the `img` tag within the HTML structure.